app/channels/iterations_channel.rb

```diff
@@ -0,0 +1,15 @@
+class IterationsChannel < ApplicationCable::Channel
```

Comment:
Can we rename this to `IterationChannel` please, and then we can use `IterationsChannel` if we want to subscribe to lists of iterations later (which we'll want to do)